opennlp-tools/src/main/java/opennlp/tools/formats/leipzig/LeipzigLanguageSampleStream.java

```diff
@@ -127,7 +128,14 @@ public LeipzigLanguageSampleStream(File leipzigFolder, final int sentencesPerSam
     this.sentencesPerSample = sentencesPerSample;
     // TODO: Use a FileFilter to make this more reliable in case there are
```

Comment:
We can remove that TODO now that this is fixing it.

Reply:
ups.. you're right